distributed/dashboard/components/worker.py

```diff
@@ -152,7 +155,14 @@ def __init__(self, worker, height=300, **kwargs):
                 alpha="alpha",
             )
 
-            hover = HoverTool(point_policy="follow_mouse", tooltips="""@hover""")
+            hover = HoverTool(
```

Comment:
Most of the hovers now have a different format, which looks like this:

![formatted_hover](https://user-images.githubusercontent.com/7526622/130133068-1ae8a887-fabb-42e8-bc1e-f52560b4d741.png)

This are the changes to make it consistent : )
```diff
diff --git a/distributed/dashboard/components/worker.py b/distributed/dashboard/components/worker.py
index e3e879c7..1840b3a7 100644
--- a/distributed/dashboard/components/worker.py
+++ b/distributed/dashboard/components/worker.py
@@ -157,12 +157,22 @@ class CommunicatingStream(DashboardComponent):
 
             hover = HoverTool(
                 point_policy="follow_mouse",
-                tooltips=[
-                    ("Transfer", "@transfer"),
-                    ("Keys", "@keys{safe}"),
-                    ("Direction", "$name"),
-                ],
+                tooltips="""
+                <div>
+                    <span style="font-size: 12px; font-weight: bold;">Transfer:</span>&nbsp;
+                    <span style="font-size: 10px; font-family: Monaco, monospace;">@transfer</span>
+                </div>
+                <div>
+                    <span style="font-size: 12px; font-weight: bold;">Keys:</span>&nbsp;
+                    <span style="font-size: 10px; font-family: Monaco, monospace;">@keys{safe}</span>
+                </div>
+                <div>
+                    <span style="font-size: 12px; font-weight: bold;">Direction:</span>&nbsp;
+                    <span style="font-size: 10px; font-family: Monaco, monospace;">$name</span>
+                </div>
+                """,
             )
+
             fig.add_tools(
                 hover,
                 ResetTool(),
```